src/Components/Steps/Expenses/Expenses.tsx

```diff
@@ -0,0 +1,323 @@
+                value={true}
+                control={<Radio />}
+                label={<FormattedMessage id="radiofield.label-yes" defaultMessage="Yes" />}
+              />
+              <FormControlLabel
+                value={false}
+                control={<Radio />}
+                label={<FormattedMessage id="radiofield.label-no" defaultMessage="No" />}
+              />
+            </RadioGroup>
+          )}
+        />
+        {fields.map((field, index) => {
+          const selectedExpenseSource = watch('expenses')[index].expenseSourceName;
+          return (
+            <Box className="expense-section-container" key={field.id}>
+              {index !== 0 && (
+                <div className="delete-button-container">
+                  <CloseButton handleClose={() => remove(index)} />
+                </div>
+              )}
+              <Stack className={index % 2 === 0 ? 'expense-section' : ''}>
+                <QuestionQuestion>
+                  {index === 0 ? (
+                    <div className="first-expense-q-padding">
+                      <FormattedMessage
+                        id="questions.hasExpenses-a"
+                        defaultMessage="What type of expense has your household had most recently?"
+                      />
+                    </div>
+                  ) : (
+                    <FormattedMessage
+                      id="expenseBlock.createExpenseBlockQuestions-questionLabel"
+                      defaultMessage="If you have another expense, select it below."
+                    />
+                  )}
+                </QuestionQuestion>
+              </Stack>
+              <FormControl
+                sx={{ m: 1, minWidth: '13.125rem', maxWidth: '100%' }}
+                error={!!errors.expenses?.[index]?.expenseSourceName}
+              >
+                <InputLabel id={`expense-type-label-${index}`}>
+                  <FormattedMessage
+                    id="expenseBlock.createExpenseDropdownMenu-expenseTypeInputLabel"
+                    defaultMessage="Expense Type"
+                  />
+                </InputLabel>
+                <Controller
+                  name={`expenses.${index}.expenseSourceName`}
+                  control={control}
+                  render={({ field }) => (
+                    <>
+                      <Select
+                        {...field}
+                        labelId={`expense-type-label-${index}`}
+                        id={`expenses.${index}.expenseSourceName`}
+                        label={
+                          <FormattedMessage
+                            id="expenseBlock.createExpenseDropdownMenu-expenseTypeSelectLabel"
+                            defaultMessage="Expense Type"
+                          />
+                        }
+                        sx={{ backgroundColor: '#fff' }}
+                      >
+                        {createExpenseMenuItems(expenseOptions)}
```

Comment:
If you make the change to `createExpenseMenuItems` above.
```suggestion
                        {createExpenseMenuItems()}
```

Reply:
The `expenseOptions` dictionary is needed to render the correct label.